js/containers/buttons/RequestApproval.jsx

```diff
@@ -5,7 +5,7 @@ var Button = require('../../components/Button.jsx');
 
 const mapStateToProps = function(state) {
 	return {
-		display: !state.approval.request_sent && state.approval.approved_by && state.git.selected_ref,
+		display: !state.deployment.is_loading && !state.deployment.data.date_requested && state.deployment.approver_id && state.git.selected_ref,
```

Comment:
maybe turn this into a func so it's easier on the eyes and brain (just for the state.deployment parts)